tests/integration/bundle/test_tls_bundle.py

```diff
@@ -21,8 +21,6 @@
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 FINOS_WALTZ = "finos-waltz"
-PGB = METADATA["name"]
```

Comment:
This was returning the name of the bundle (the field that was originally changed in this PR).